time-based-packet-transmission-offload.md

```diff
@@ -153,6 +178,24 @@ typedef enum _NDIS_TXTIME_SUPPORT_FLAGS {
 
 ```
 
+To obtain the `NDIS_TCP_IP_TXTIME_NET_BUFFER_LIST_INFO` structure, a driver should call the `NET_BUFFER_LIST_INFO` macro with an _Id of `TcpIpTxTimeNetBufferListInfo`
+```C
+typedef struct _NDIS_TCP_IP_TXTIME_NET_BUFFER_LIST_INFO {
+  struct {
+    ULONG IsIPv4 : 1;
+    ULONG IsIPv6 : 1;
+    ULONG IsTcp : 1;
+    ULONG IsUdp : 1;
+    ULONGLONG TxTime : 64;
+    ULONGLONG TimeDelta : 64;
+  } Transmit;
```

Comment:
Pacing only makes sense for TX. No need to wrap everything in `Transmit.` It's implicit.

Reply:
I'm investigating whether TransmitComplete is required for TPTO....

Reply:
Only TxTime and TimeDelta should be enough